src/hooks/analytics/segment-adapter.ts

```diff
@@ -40,6 +47,37 @@ export class SegmentAdapter {
     this.id = segmentId
   }
 
+  /**
+   * Returns anonymous id to identify and track chectl events in segment
+   */
+  static getAnonymousId(): string {
+    const anonymousIdPath = path.join(os.homedir(), '.redhat', 'anonymousId')
+
+    if (fs.existsSync(anonymousIdPath)) {
+      return fs.readFileSync(anonymousIdPath, 'utf8')
+    } else {
+      const anonymousId = v4()
+      if (!fs.existsSync(anonymousIdPath)) {
+        fs.mkdirSync(path.join(os.homedir(), '.redhat'))
```

Comment:
We should catch error here.

Reply:
ok
@tolusha  In case of catch(e) {} Should we print a message there or don't nothing?